rest/reference/exchanges.go

```diff
@@ -0,0 +1,44 @@
+package reference
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/polygon-io/client-go/rest/client"
+	"github.com/polygon-io/client-go/rest/models"
+)
+
+// ListExchangesIter is an iterator for the ListExchanges method.
+type ListExchangesIter struct {
```

Comment:
i'm pretty sure this isn't a list/iterator method (the response would include a next URL if it was)

Reply:
Ah I got tunnel visioned because it was in a results array. I'll fix it.